create_services.yml

```diff
@@ -0,0 +1,20 @@
+- hosts: local
+  gather_facts: False
+
+  pre_tasks:
+    - import_tasks: tasks/check_configuration.yml
+
+  tasks:
+    - name: Display playbook name
+      debug: msg="==== Starting create_services playbook ===="
+      tags: deploy
+
+    - import_tasks: tasks/set_vars.yml
+
+    - include_tasks: tasks/run_tasks_for_apps.yml
+      vars:
+        tasks:
+          - get_objects_for_app
+          - create_static_services
```

Comment:
you create only static services, you should maybe rename this file in `create_static_services` ?

Reply:
Thanks for your feedbacks !

I tried to do that in a first place, but it triggered an error.
The only reason I see is because there is already a file named `create_static_services.yml` in the tasks folder.